doc/api/tracing.md

```diff
@@ -80,7 +80,7 @@ string that supports `${rotation}` and `${pid}`:
 node --trace-event-categories v8 --trace-event-file-pattern '${pid}-${rotation}.log' server.js
 ```
 
-Starting with Node.js 10.0.0, the tracing system uses the same time source
+Starting with Node.js v10.0.0, the tracing system uses the same time source
```

Comment:
We avoid the `v` prefix in prose, see https://github.com/nodejs/node/blob/master/doc/guides/doc-style-guide.md.

> * When referring to a version of Node.js in prose, use _Node.js_ and the version
  number. Do not prefix the version number with _v_ in prose. This is to avoid
  confusion about whether _v8_ refers to Node.js 8.x or the V8 JavaScript
  engine.
>   * OK: _Node.js 14.x_, _Node.js 14.3.1_
>   * NOT OK: _Node.js v14_